### ISSUE TITLE
[Bug] Themed / Monochrome app icon not working properly on GrapheneOS

### ISSUE THREAD

**Issue by sebastien46** [toasterofbread/spmp#95]
default camera app for reference:
![IMG_20230723_210303 1](https://github.com/toasterofbread/spmp/assets/57289288/07a59338-6660-4c07-bcc5-8d4113507e43)

_Originally posted by @sebastien46 in https://github.com/toasterofbread/spmp/issues/74#issuecomment-1646933717_

**Comment by toasterofbread** [comment]
Hmm, I don't know why that would happen. I assume all of your other apps are displaying correctly? Could you give an example of an open-source app that displays the monochrome icon properly for you so that I can see how they implement it?

Also, just so I can test it in an emulator, what version of GrapheneOS are you using?

**Comment by sebastien46** [comment]
Yes, all of my other apps that implement monochrome icons work properly.
For an example of an open-source app that works for me, here's [Auxio](https://github.com/OxygenCobalt/Auxio).
I use the latest GrapheneOS version on Android 13 (GPixel7).

**Comment by toasterofbread** [comment]
Thanks, I've taken a look at Auxio and there's nothing obvious that I'm doing differently so I'll do some testing in an emulator once I'm back home in a few days.

**Comment by toasterofbread** [comment]
Sorry about the delay. I've tried running GrapheneOS using an AVD emulator but I can't seem to get it working. I'll still keep trying to test this out (I should be able to find an old phone somewhere that I can flash with GrapheneOS).

Just to be sure, you were using an APK that was built after [the commit](https://github.com/toasterofbread/spmp/commit/8e2165b3ed04388d2b7f6d86c5d42669d9a60a20) was added, right? (Such as from [here](https://github.com/toasterofbread/spmp/actions/runs/5593416030))
